scripts/compare-testsuite-log.py

```diff
@@ -369,17 +369,20 @@ def is_result_valid(log_path: str):
                     return False
         return True
 
-
-def main():
-    args = parse_arguments()
-    if not is_result_valid(args.previous_log):
+def run(previous_hash: str, previous_log: str, current_hash: str, current_log: str, output_markdown: str):
+    if not is_result_valid(previous_log):
         raise RuntimeError("Previous Log doesn't include Summary of the testsuite")
```

Comment:
Is there a way we could propagate this error to the summarizer so the github issue can include it as a reason for why it didn't work? (That way we don't have to dig through the logs to find this)

Reply:
Error is now added to a new column called additional info